src/views/Scope.vue

```diff
@@ -60,6 +60,7 @@ implemented visualizers.
     import CanvasArea from '../components/CanvasArea.vue'
     import BundleManager from '../components/BundleManager.vue'
     import VISUALIZERS from '../visualizers/visualizers'
+    import WORKBENCH from '../workbench/visualizers'
```

Comment:
The identifier WORKBENCH should be changed to one that indicates the connection with visualizers

Reply:
It's now `visualizers-workbench`.